kavm/src/kavm/pyk_utils.py

```diff
@@ -109,7 +127,8 @@ def __init__(
         self._store: Dict[Any, Any] = {}
         self._unit_klabel = unit_klabel.name if isinstance(unit_klabel, KLabel) else unit_klabel
         assert self._unit_klabel.endswith('CellMap')
-        self._item_cell_name = '<' + re.sub('CellMap$', '', self._unit_klabel).strip('.').lower() + '>'
+        item_name = re.sub('CellMap$', '', self._unit_klabel).strip('.')
```

Comment:
This was needed because the renaming scheme before was assuming that the cell name was a single word, and so converting the whole string to lowercase was equivalent to converting from upper to lower camel case.